### ISSUE TITLE
Add tests to `my-example-dapp`

### ISSUE THREAD

**Issue by Dhaiwat10** [FuelLabs/fuels-ts#643]
### Motivation

Before we start work on #568, after my discussion with Anderson on Slack we think it would be a good idea to add some tests to `my-example-dapp` as a POC. 

eg. tests driving the clicks/inputs and validating the UI/DOM states (has the counter changed to X? etc.)

**Comment by Dhaiwat10** [comment]
The PoC is ready here: https://github.com/Dhaiwat10/my-fuel-dapp. Now figuring out the best way to add this to our CI setup in the `fuels-ts` repo

**Comment by Dhaiwat10** [comment]
We have decided to put UI/React tests on hold for now so marking this as blocked.

**Comment by Dhaiwat10** [comment]
Closing this issue since this example has been deprecated.
